packages/esm-patient-forms-app/src/forms/forms-table.component.tsx

```diff
@@ -33,10 +40,21 @@ interface FormsTableProps {
   isTablet: boolean;
   handleSearch: (search: string) => void;
   handleFormOpen: (form: Form, encounterUuid: string) => void;
+  totalItems: number;
 }
 
-const FormsTable = ({ tableHeaders, tableRows, isTablet, handleSearch, handleFormOpen }: FormsTableProps) => {
+const FormsTable = ({
+  tableHeaders,
+  tableRows,
+  isTablet,
+  handleSearch,
+  handleFormOpen,
+  totalItems,
+}: FormsTableProps) => {
   const { t } = useTranslation();
+  const { pageSize, currentPage, setPageSize, setCurrentPage } = useFormsContext();
+  const pageSizes = [50, 100, 200];
```

Comment:
I think something more granular would be better here:

```suggestion
  const pageSizes = [10, 20, 30, 40, 50];
```

Reply:
The goal is to make it easier for people who have more than 150 forms to easily navigate, and since the default rest response is 50, i chose 50, but i can make it configurable